### DISCUSSION TITLE
Looking for lcms2.dll

### DISCUSSION THREAD

**Discussion by RobLaFreniere** [saucecontrol/PhotoSauce#128] [Q&A] [answered]
I recently updated my project to use MagicScaler  0.14 and ran a test suite that loads a hundred or so photos and saw a single failure  to load "lcms2.dll".  Is that something I need to source elsewhere or should it be part of the distribution?

**Comment by saucecontrol** [comment, marked as answer]
I assume this was from some external profiling tool and not an exception from the library itself?

MagicScaler will look for lcms2 and use it if found.  On Windows, it's not necessary, as the Windows Color System will handle conversions as a fallback.  On Linux, LittleCMS is required for some conversions (mostly CMYK), but you should get an exception directing you to install it from your package manager in that case.
